vendor/BUILD

```diff
@@ -319,12 +319,7 @@ filegroup(
         "//vendor/github.com/pkg/errors:all-srcs",
         "//vendor/github.com/pmezard/go-difflib/difflib:all-srcs",
         "//vendor/github.com/pquerna/cachecontrol:all-srcs",
-        "//vendor/github.com/prometheus/client_golang/prometheus:all-srcs",
-        "//vendor/github.com/prometheus/client_model/go:all-srcs",
-        "//vendor/github.com/prometheus/common/expfmt:all-srcs",
-        "//vendor/github.com/prometheus/common/internal/bitbucket.org/ww/goautoneg:all-srcs",
-        "//vendor/github.com/prometheus/common/model:all-srcs",
-        "//vendor/github.com/prometheus/procfs:all-srcs",
+        "//vendor/github.com/prometheus:all-srcs",
```

Comment:
was this hand-edited? won't the hack/update-bazel script undo these changes?

Reply:
After I added BUILD file with visibility rules to `vendor/github.com/prometheus` hack/update-bazel script added `all-srcs` rules to it and replaced it `vendor/BUILD`

Reply:
Thanks to @munnerz suggestion I updated `hack/update-bazel.sh` to set visibility.